README.md

```diff
@@ -59,6 +59,30 @@ Use `^C` to stop the container and `docker-compose down` to remove the last used
 Many TypeScript and static file changes can be made without re-building the image,
 but as necessary, rebuild the image using Step 1.
 
+#### Step By Step
+
+Explicit steps for setting up CCM in a development environment.
+
+1. `ngrok http 4000`
+1. `curl --silent http://127.0.0.1:4040/api/tunnels | jq -r '.tunnels[0].public_url'`
+   Copy only the hostname, not the `http`/`https` scheme prefix.
+1. `cp config/lti_dev_key_sample.json config/lti_dev_key.json`
+1. `mvim config/lti_dev_key.json`
+   Replace all occurrences of `{ccm_app_url}` with the hostname copied in the earlier step.
+1. Go to Canvas "Developer Keys" management page at (https://umich.instructure.com/accounts/1/developer_keys).  Click the button for "+ Developer Key", then "+ LTI Key".
+1. When the "Key Settings" modal appears, select "Paste JSON" from the "Method" menu in the "Configure" section.  Then copy the contents of your `config/lti_dev_key.json` and paste it into the "LTI 1.3 Configuration" field.
+1. Save LTI Key.
+1. Edit LTI Key, use "Manual Entry" under the "Method" menu to access "LTI Advantage Services" under the "Configuration" section.  Enable all options, then click the "Save" button.
+   1. Note that after saving, using "Paste JSON" under the "Method" menu, will show the corresponding JSON for the changes.  Enabling the LTI Advantage options adds them to the `scopes` key of the JSON.
```

Comment:
We don't have to do this yet, since we're not using these services.

Reply:
Sure.  Would you prefer that this be removed?

Reply:
Is there any reason to include it yet? I think if we needed an LTI Advantage Service, I'd include it as a scope in the sample developer key JSON.